resources/js/Pages/Results.vue

```diff
@@ -136,251 +136,205 @@
     </div>
 </template>
 
-<script lang="ts">
-    import { PropType } from 'vue';
-    import { useStore } from '../store';
-    import isEmpty from 'lodash/isEmpty';
-    import { Head as InertiaHead } from '@inertiajs/inertia-vue3';
-    import { CdxButton, CdxIcon, CdxDialog, CdxMessage, CdxCheckbox } from "@wikimedia/codex";
-    import { cdxIconInfo, cdxIconArrowPrevious } from '@wikimedia/codex-icons';
-    import LoadingOverlay from '../Components/LoadingOverlay.vue';
-    import MismatchesTable from '../Components/MismatchesTable.vue';
-    import Mismatch, {ReviewDecision, LabelledMismatch} from '../types/Mismatch';
-    import User from '../types/User';
-    import { defineComponent } from 'vue';
-    import axios from 'axios';
-
-    // Run it with compat mode
-    // https://v3-migration.vuejs.org/breaking-changes/v-model.html
-    CdxCheckbox.compatConfig = {
-      ...CdxCheckbox.compatConfig,
-      COMPONENT_V_MODEL: false,
-    };
-    interface MismatchDecision {
-        id: number,
-        item_id: string,
-        review_status: ReviewDecision,
-        previous_status: ReviewDecision
+<script setup lang="ts">
+import { useStore } from '../store';
+import isEmpty from 'lodash/isEmpty';
+import { Head as InertiaHead } from '@inertiajs/inertia-vue3';
+import { CdxButton, CdxIcon, CdxDialog, CdxMessage, CdxCheckbox } from "@wikimedia/codex";
+import { cdxIconInfo, cdxIconArrowPrevious } from '@wikimedia/codex-icons';
+import LoadingOverlay from '../Components/LoadingOverlay.vue';
+import MismatchesTable from '../Components/MismatchesTable.vue';
+import Mismatch, {ReviewDecision, LabelledMismatch} from '../types/Mismatch';
+import User from '../types/User';
+import { Ref, computed, onMounted, ref } from 'vue';
```

Comment:
Are we sure we need both Ref and ref?

Reply:
yes `Ref` is a type, `ref` is a function :)

Reply:
It'll avoid such misunderstandings as the one I had, so i'm for it of course :grimacing:

Reply:
Oh that's true. Thanks for the reminder! Will update.